funnel/utils/markdown/mdit_plugins/abbr.py

```diff
@@ -0,0 +1,173 @@
+"""
+Markdown-it-py plugin to introduce <abbr> markup for defined abbreviations.
+
+Ported from javascript plugin markdown-it-abbr.
+"""
+
+import re
+import unicodedata
+
+from markdown_it import MarkdownIt
+from markdown_it.rules_block import StateBlock
+from markdown_it.rules_inline import StateInline
+from markdown_it.token import Token
+
+__all__ = ['abbr_plugin']
+
+ASTERISK_CHAR = 0x2A  # ASCII value for `*`
+OPEN_SQBR_CHAR = 0x5B  # ASCII value for `[`
+CLOSE_SQBR_CHAR = 0x5D  # ASCII value for `]`
+BACKSLASH_CHAR = 0x5C  # ASCII value for `\`
+COLON_CHAR = 0x3A  # ASCII value for `:`
```

Comment:
I'm wondering if we should define these constants as `ord('*')`. It's a function call, but it's self-validating unlike the comment.